src/main/java/be/seeseemelk/mockbukkit/scheduler/BukkitSchedulerMock.java

```diff
@@ -30,6 +29,7 @@
 	private AtomicInteger asyncTasksRunning = new AtomicInteger();
 	private AtomicReference<Exception> asyncException = new AtomicReference<>();
 	private int asyncTasksQueued = 0;
+	private ConcurrentHashMap<Integer, ScheduledTask> runners = new ConcurrentHashMap<>();
```

Comment:
It is generally best to declare collections using their base type.
It makes it easier to swap out the implementation later on.
```suggestion
	private ConcurrentMap<Integer, ScheduledTask> runners = new ConcurrentHashMap<>();
```